search/tutorials/buildsearchbar.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: "Docs Search Quickstart"
+description: "A quick guideline to crawl docs and create a functional search component"
+---
+
+---
+
+### 1. Project Setup
+
+Let's create a new Next.js application and install the related packages:
+
+```shell
+npx create-next-app@latest search-docs-app
+cd search-docs-app
+npm install @upstash/search @upstash/search-ui lucide-react
+```
+
+---
+
+### 2. Add Environment Variables
+
+Find the environment variables from your database dashboard and add them to your `.env` file:
```

Comment:
Something like `Create a Upstash Search Database if you already don't have one and ...` would be better I think